requirements/base.txt

```diff
@@ -1,20 +1,12 @@
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Base Requirements.                                                          #
-attrs>=21.2.0
+# Base Algo Requirements.                                                          #
 natsort>=6.0.0
 nbmake
-networkx>=2.6,<=2.8.0
-numpy>=1.21.0,<=1.23.4  # np.bool was removed in 1.24.0 which was used in openvino runtime
-omegaconf>=2.1.1
-opencv-python>=4.5
 prettytable
 protobuf>=3.20.0
-pymongo==3.12.0
 pyyaml
-scikit-learn==0.24.*
-Shapely>=1.7.1,<=1.8.0
 imagesize==1.4.1
-datumaro==0.4.0.1
+datumaro==1.0.0rc1
```

Comment:
I'm not sure Datumaro1.0.0 perfectly fits with the previous version. However, ,maybe there is no problem. let's check the test results.

Reply:
Yeah, if there is some problem, we can rollback as I separated api requirement (w/o datumaro) so that there will be no version conflict. Anyway, 1.0.0rc1 is already integrated to Geti develop, though.